libraries/Microsoft.Bot.Builder.Dialogs.Adaptive/Actions/HttpRequest.cs

```diff
@@ -332,6 +339,13 @@ public override async Task<DialogTurnResult> BeginDialogAsync(DialogContext dc,
                     requestResult.Content = content;
                     break;
 
+                case ResponseTypes.BinaryData:
```

Comment:
>BinaryData [](start = 35, length = 10)

You need to add a unit test for this.

Reply:
> > BinaryData [](start = 35, length = 10)
> 
> You need to add a unit test for this.

Hi Tom, do I need to add the UT under Microsoft.Bot.Builder.Dialogs.Adaptive.Tests? I think I need a mock http client to handle all fake data... Am I right